src/components/screens/SearchView.vue

```diff
@@ -7,8 +7,9 @@
       <div class="mavedb-search-form">
         <span class="p-input-icon-left">
           <i class="pi pi-search" />
-          <InputText v-model="searchText" ref="searchTextInput" type="search" class="p-inputtext-sm" placeholder="Search" />
-          <Button v-if="searchText && searchText.length > 0" class="mavedb-search-clear-button p-button-plain p-button-text" @click="clear"><i class="pi pi-times"/></Button>
+          <InputText v-model="searchText" ref="searchTextInput" type="search" class="p-inputtext-sm" placeholder="Search" /> &nbsp;
+          <!--<Button v-if="searchText && searchText.length > 0" class="mavedb-search-clear-button p-button-plain p-button-text" @click="clear"><i class="pi pi-times"/></Button>-->
```

Comment:
this change seems unrelated and should probably be pulled out to a separate frontend change if you want to make it.

Reply:
Sure! I think it'll be better and clearer for everyone. I'll create another pull request for this feature and change this back in this branch.